homeassistant/bootstrap.py

```diff
@@ -552,6 +554,14 @@ def enable_logging(hass: core.HomeAssistant, verbose: bool=False,
 
         async_handler = AsyncHandler(hass.loop, err_handler)
 
+        @core.callback
+        def _async_close_handler(event):
+            """Close async handler on stop."""
+            async_handler.close()
+
+        hass.bus.async_listen_once(
+            EVENT_HOMEASSISTANT_STOP, _async_close_handler)
```

Comment:
Not in all case. Some time it does not and in other case it does. We need to call logging.shutdown() and it will close in all case, but we can not call this.